sass/chosen.scss

```diff
@@ -1,6 +1,35 @@
 $chosen-sprite: url('chosen-sprite.png') !default;
 $chosen-sprite-retina: url('chosen-sprite@2x.png') !default;
 
+$bg-color: #fff;
```

Comment:
All variable names should start with `chosen-` to avoid conflicts with other styles for people importing the Scss in their own files.